service-api/pom.xml

```diff
@@ -156,6 +157,11 @@
             <version>2.2.19</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${guava.version}</version>
```

Comment:
I do not see usage of this dependency
Why did you add it?

Reply:
We are using this dependency in the LoginAttemptServiceImpl class. It is declared as a private final field LoadingCache<String, Integer> and initialized in the constructor.